CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 0.7.2
+ - Fix OutOfMemory handling in Solr init script. This is a bug in Solr 5.2.1
+ and is fixed in 5.5.1+. We have backported the fix here. The arguments were
+ being passed in, in the wrong order and ignored.
+
```

Comment:
Would be good to reference this https://issues.apache.org/jira/browse/SOLR-8145 for future travellers.